content/quiz_meets.md

```diff
@@ -1,6 +1,13 @@
 # Quiz Meets
+The tournament brackets are based on three things:   A preliminary round, an elimination round (at the discretion of the meet director), and championship quizzes.
 
 ## Preliminary Rounds
+The teams will all receive team points as follows:
+- First place in a quiz:  10 points, plus 1 point for every 10 points over 100
```

Comment:
I decided not to define "Raw Points" and "Calculated Team Points"

Reply:
WFM. But I'm thinking this needs to be English-Script and in the scoring section?